docs/marketing/treasurer.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Treasurer
```

Comment:
I don't think we need YAML content for adding roles. Meta data should be kept for blogs.